pymc_experimental/utils/pivoted_cholesky.py

```diff
@@ -0,0 +1,68 @@
+try:
+    import torch
+    from gpytorch.utils.permutation import apply_permutation
+except ImportError as e:
+    # print(
```

Comment:
instead of a print use `raise ImportError(...)`

Reply:
definitely, also consider `raise ImportError(...) from e` https://stackoverflow.com/questions/24752395/python-raise-from-usage